src/uk/gov/hmcts/contino/Consul.groovy

```diff
@@ -8,12 +8,14 @@ class Consul {
 
   def steps
   def subscription
+  def environment
   def az
   private consulApiAddr
 
-  Consul(steps) {
+  Consul(steps, environment) {
```

Comment:
can you check github for any uses of this class outside the library?

Reply:
Thanks, There is one place where they are using. Should i make it backward compatible with default env as preview? 
https://github.com/hmcts/probate-intestacy-frontend/blob/56d7a9350059232e9213b358084f779ad91f17b9/Jenkinsfile_CNP#L76

Reply:
yup, nice work